Xamarin.Forms.Platform.iOS/Material/MaterialStepperRenderer.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.ComponentModel;
+using CoreGraphics;
+using MaterialComponents;
+using UIKit;
+using Xamarin.Forms;
+using MButton = MaterialComponents.Button;
+
+[assembly: ExportRenderer(typeof(Xamarin.Forms.Stepper), typeof(Xamarin.Forms.Platform.iOS.Material.MaterialStepperRenderer), new[] { typeof(VisualRendererMarker.Material) })]
+
+namespace Xamarin.Forms.Platform.iOS.Material
+{
+	public class MaterialStepperRenderer : ViewRenderer<Stepper, MaterialStepper>
+	{
+		ButtonScheme _buttonScheme;
+
+		public MaterialStepperRenderer()
+		{
+			VisualElement.VerifyVisualFlagEnabled();
+		}
+
+		protected override void Dispose(bool disposing)
+		{
+			if (Control is MaterialStepper control)
+			{
+				control.DecrementButton.TouchUpInside -= OnStep;
+				control.IncrementButton.TouchUpInside -= OnStep;
+			}
+
```

Comment:
Should this also be disposing _buttonScheme?

Reply:
Fixed.